tools/scylla-nodetool.cc

```diff
@@ -3974,11 +4040,12 @@ repaired. Typically, this subset of data is replicated on many nodes in
 the cluster, often all, and the repair process syncs between all the
 replicas until the master data subset is in-sync.
 
-Note that nodetool repair repairs only vnodes-based keyspaces.
-
 To repair all of the vnodes-based data in the cluster, you need to run a repair on
 all of the nodes in the cluster, or let ScyllaDB Manager do it for you.
 
+Note that nodetool repair repairs only vnodes-based keyspaces. To repair
+tablet-based keyspaces use nodetool tablet-repair.
```

Comment:
patch-organization nitpick: if you know the final state of this hunk is here, add it to here in the patch which introduces it, so later patch just modifies it, it doesn't have to move it.